### PR TITLE
Add EIP: To The Moon—10 Minute Blocks

### PR DESCRIPTION
**Pull request by Pandapip1**

### CODE REVIEW

File: EIPS/eip-6811.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+created: 2023-04-01
+---
+
+## Abstract
+
+This EIP makes a minimal number of changes to allow Ethereum to be used on the moon and other potentially habitable bodies in Earth's solar system. It changes the time between blocks, the per-block validator reward, and the number of blocks per epoch.
+
+## Motivation
+
+It is impossible for Ethereum to literally "go to the moon" due to a limitatio in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) is might cause issues with synchronization, considering the 12-second timer between blocks. The validators would find themselves slashed on the terrestrial chain, and validating their own fork.
```
Comment (SamWilsn) [inline EIPS/eip-6811.md:19]:
```suggestion
It is impossible for Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would find themselves slashed on the terrestrial chain, and validating their own fork.
```

File: EIPS/eip-6811.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+created: 2023-04-01
+---
+
+## Abstract
+
+This EIP makes a minimal number of changes to allow Ethereum to be used on the moon and other potentially habitable bodies in Earth's solar system. It changes the time between blocks, the per-block validator reward, and the number of blocks per epoch.
+
+## Motivation
+
+It is impossible for Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would find themselves slashed on the terrestrial chain, and validating their own fork.
```
Comment (g11tech) [inline EIPS/eip-6811.md:19]: validators won't be slashed per se but they would `leak` and be force exited in each other's forks
```suggestion
It is impossible for Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would eventually be ejected on the terrestrial chain after leaking. If however a substantial number of validators are displaced (think 1/3), they might follow their own fork and would eventually eject the terrestrial to finalize their own chain.
```

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
Hey @SamWilsn, I would love this to be merged.

**Comment by SamWilsn** [comment]
Just to be doubly extra sure, you want to continue the standardization process, and plan to bring this EIP up on the AllCoreDevs calls as a serious proposal? I only ask because of the date this was opened.

**Comment by Pandapip1** [comment]
> Just to be doubly extra sure, you want to continue the standardization process, and plan to bring this EIP up on the AllCoreDevs calls as a serious proposal? I only ask because of the date this was opened.

Yes, I really do. The date just gives me license to use more humorous language.

File: EIPS/eip-asdf.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+title: Ethereum To The Moon!
```
Comment (SamWilsn) [inline EIPS/eip-asdf.md:2]: The title of this proposal isn't descriptive enough. Perhaps:

```suggestion
title: To The Moon—10 Minute Blocks
```

Or just:

```suggestion
title: Increase block time to 10 minutes
```

File: EIPS/eip-asdf.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+description: Makes a minimal number of changes that allow Ethereum to be used on the moon
```
Comment (SamWilsn) [inline EIPS/eip-asdf.md:3]:
```suggestion
description: Increases the block time to facilitate finality over cosmic distances
```

File: EIPS/eip-asdf.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+created: 2023-04-01
+---
+
+## Abstract
+
+This EIP makes a minimal number of changes to allow Ethereum to be used on the moon and other potentially habitable bodies in Earth's solar system. It changes the time between blocks, the per-block validator reward, and the number of blocks per epoch.
+
+## Motivation
+
+It is impossible for Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would eventually be ejected on the terrestrial chain after leaking. If however a substantial number of validators are displaced (think 1/3), they might follow their own fork and would eventually eject the terrestrial to finalize their own chain.
```
Comment (SamWilsn) [inline EIPS/eip-asdf.md:18]:
```suggestion
It is impossible for today's Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would eventually be ejected on the terrestrial chain after leaking. If however a substantial number of validators are displaced (think 1/3), they might follow their own fork and would eventually eject the terrestrial to finalize their own chain.
```

File: EIPS/eip-asdf.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+* The time between blocks MUST be changed from 12 seconds to 10 minutes (50x longer).
```
Comment (SamWilsn) [inline EIPS/eip-asdf.md:24]:
```suggestion
* The time between blocks MUST be changed from 12 seconds to 600 seconds (10 minutes).
```

File: EIPS/eip-asdf.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+* The per-block validator reward MUST be multiplied by 50
+* The number of blocks per epoch MUST be reduced from 4 to 2
+
+## Rationale
+
+* The block gas limit is multiplied by 50 to compensate for the time between blocks being multiplied by fifty.
```
Comment (SamWilsn) [inline EIPS/eip-asdf.md:30]:
```suggestion
* The block gas limit is multiplied by fifty to compensate for the time between blocks being multiplied by fifty.
```

File: EIPS/eip-asdf.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+* The per-block validator reward is also multiplied by 50 to copensate for the time between blocks being multiplied by fifty.
```
Comment (SamWilsn) [inline EIPS/eip-asdf.md:31]:
```suggestion
* The per-block validator reward is also multiplied by fifty to compensate for the time between blocks being multiplied by fifty.
```

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**